lib/tasks/valid_test_data_generator.rb

```diff
@@ -151,4 +153,80 @@ def random_or_nil_trn
       TRNGenerator.next
     end
   end
+
+  class NPQLeadProviderPopulater
+    class << self
+      def call(name:, total_schools: 10, participants_per_school: 10)
+        new(name: name).call(total_schools: total_schools, participants_per_school: participants_per_school)
+      end
+    end
+
+    def call(total_schools: 10, participants_per_school: 100)
+      generate_new_schools(count: total_schools, participants_per_school: participants_per_school)
+    end
+
+  private
+
+    attr_reader :lead_provider
+
+    def initialize(name:)
+      @lead_provider = ::NPQLeadProvider.find_or_create_by!(name: name)
+    end
+
+    def generate_new_schools(count:, participants_per_school:)
+      count.times { create_fip_school_with_cohort(urn: SchoolURNGenerator.next, participants_per_school: participants_per_school) }
+    end
+
+    def find_or_create_participants(school:, number_of_participants:)
+      generate_new_participants(school: school, count: number_of_participants)
+    end
+
+    def generate_new_participants(school:, count:)
+      while count.positive?
+        count -= 1
+        create_participant(school: school, accepted: count.even?)
```

Comment:
`accepted: count.even?` Other variables are deliberately using a representative sampling. Do we really expect a 50% rejection ratio?

Reply:
Technically speaking the rejection ratio is 0%. The other 50% are pending.

I have no idea what we are expecting. When we know for sure, we can code it in. In the meantime, deterministic 50% seems better than non-deterministic percentages, because at least we can use it to comfortably seed our local db and review app db with something that has some pending and some approved applications.